### ISSUE TITLE
UI: Add Resources to 'Learn Git & GitHub' Page

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#351]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [x] User Interface
- [ ] Feature Request

## Description

We need to add the 'Learn Git & GitHub' resources from the repository to the 'Learn Git & GitHub' page and ensure that they exist in separate boxes. It can look like this:

<p align="center">
  <img src="https://github.com/Sriparno08/Openpedia/assets/89148144/3df0d084-9ec2-4723-a6a4-93abf4d39c10">
</p>

**Comment by Bibek478** [comment]
I would like work on this under JWOC24

**Comment by Sriparno08** [comment]
You're already assigned to another issue. Once your PR for it is merged, you will be considered for this issue. If anyone asks to be assigned before your PR is merged, he/she will be assigned.

**Comment by Sriparno08** [comment]
You're assigned.

**Comment by Bibek478** [comment]
Could you please explain a bit more what to fix here

**Comment by Sriparno08** [comment]
Sure. We want to add the resources under 'Learn Git & GitHub' header in this repository to the 'Learn Git & GitHub' page in the website. The structure can look like this:

![image](https://github.com/Sriparno08/Openpedia/assets/89148144/6f76d77b-5a65-4594-8036-fce6913920f4)

**Comment by Bishal-35** [comment]
I would like to work on it under JWoC'24
